sources/api/services/assistant/Assistant.ts

```diff
@@ -126,3 +130,35 @@ export function findBestSlots(groupCalendar: Calendar, calendars: Calendar[], du
     }
     return normaliseSlots(slots, min_score, max_score).sort((a: TimeSlot, b: TimeSlot) => b.score - a.score);
 }
+
+function getAttendableEvents(calendars: Calendar[], events: Event[], min_time: Date, max_time: Date): Event[] {
+    const attendable_events: Event[] = [];
+
+    for (const event of events) {
+        if (event.startTime >= min_time && event.endTime <= max_time
+            && slotAvailableAll(calendars, event.startTime, event.endTime, TRAVEL_TIME)) {
+            attendable_events.push(event);
+        }
+    }
+    return attendable_events;
+}
+
+function getSuggestionScore(user: User, event: Event): number {
+    return 1;
+}
+
+export function getEventSuggestions(user: User, calendars: Calendar[], events: Event[],
+        min_time: Date, max_time: Date)
+        : EventSuggestion[] {
+    const attendable_events = getAttendableEvents(calendars, events, min_time, max_time);
+
+    const events_scores: EventSuggestion[] = [];
```

Comment:
`eventSuggestion`

Also, try to use `camelCase` (if we did use `snake_case` in the past, we should not have 😉)